cmd/controller/main.go

```diff
@@ -18,6 +18,7 @@ package main
 
 import (
 	// The set of controllers this controller process runs.
+	certificate "knative.dev/control-protocol/pkg/certificates/reconciler"
```

Comment:
Use certificates?

Reply:
I used singular here as other controller libs are singular and the following line (L#38 to L#46) could be consistent:

```go
var ctors = []injection.ControllerConstructor{
        configuration.NewController,
        labeler.NewController,
        revision.NewController,
        route.NewController,
        serverlessservice.NewController,
        service.NewController,
        gc.NewController,
        nscert.NewController,
        certificate.NewControllerFactory(networking.ServingCertName),
}
```